package.json

```
@@ -8,13 +8,13 @@
     "test:schema": "ota validate --schema-only",
     "test:modified": "ota validate --modified",
     "start": "ota track",
-    "start:api": "ota serve",
+    "start:collection-api": "ota serve",
     "start:schedule": "npm run start -- --schedule",
     "start:modified": "git diff -z --name-only HEAD $(git merge-base HEAD origin/main) -- ./declarations/ | tr '\\0' '\\n' | grep -v .history | grep -v .filters | sed 's/declarations\\///g' | sed 's/.json//g' | sed 's/.*/\"&\"/' | xargs -r npm run start -- --services",
     "dataset": "ota dataset",
     "dataset:schedule": "npm run dataset -- --publish --remove-local-copy --schedule"
   },
   "dependencies": {
-    "@opentermsarchive/engine": "~2.0.1"
+    "@opentermsarchive/engine": "~2.2.1"
```

Why don't we use `^` instead of `~`? I trust our semver publishing 🙂 
```suggestion
    "@opentermsarchive/engine": "^2.2.1"
```

(if accepted, this will also need regenerating the package lock file)

---

Well catch!